src/components/MainMenu.module.css

```diff
@@ -0,0 +1,275 @@
+
+
+.MainMenu{
+  display: flex;
+  flex-direction: column;
+height: 100%;
+border-radius:50px;
+}
+
+
+
+.title{
+  font-size: 30px;
+ text-align: center;
+ font-family: "Streamster";
+ color:#F94E9B;
+ text-shadow: 1px 1px 2px #fdc500;
+ 
+}
+
+
+.logo {
+text-align: center;
+font-family: "RoadRage";
+color: #fdc500;
+font-size: 20px;
+text-shadow: 2px 2px 2px #5c223b
+}
+
+.menuItems{
+  display: flex;
+  justify-content: space-around;
+  margin-bottom: 10px;
+  
+}
+
+ 
+a {
+  font-family: "Neon";
+  color: #F94E9B;
+  text-decoration: none;
+  text-transform: uppercase;
+
+}
+
+.itemWrapper{
+  display: inline-block;
+  overflow: hidden;
+  font-size: 15px;
+width: 150px;
+text-align:center;
+border-radius:5px;
+transition: all .5s;
+}
+
+
+.chosen{
+  background-color:#fdc500;
+  transform:scale(1.3);
+  transition: background-color 1s;
+}
+
+
+
+.itemWrapper:hover {
+  transform: scale(1.3) ;
+  transition: transform .2s ease;
+}
+
+
+
+
+
+.slideContainer{
+
+  padding: 40px 0 0;
+  display: flex;
+
+  justify-content: space-around;
+  font-size: 20px;
+  font-family: "Neon";
+}
+
+.effectContainer{
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  color: #F94E9B;
+}
+
+.inputWrapper{
+  width:max-content;
+}
+.effectContainer span{
+  display: block;
+  width:150px;
+  text-align: center;
+  height: 100%;
+}
+
+.slideContainer input{
+  margin-left: 10px;
+  cursor: pointer;
+}
+
+
+
+
+
+.enableSound{
+background-color:#fdc500;
+text-align: center;
+width: 40px;
+padding:1px 5px ;
+border-radius:5px;
+margin-left: 10px;
+cursor: pointer;
+}
+
+
+
+.otherSettings{
+  margin-top: 20px;
+  margin-right:auto;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  width: 100%;
```

Comment:
it is better to make the indents the same

Reply:
done